src/Pages/HomePage.tsx

```diff
@@ -1,188 +1,112 @@
 
 export default function HomePage() {
+	const dispatch = useAppDispatch()
+	const categories = useSelector(categoriesSelector)
+	const status = useSelector(statusSelector)
+	useEffect(() => {
+		dispatch(fetchCategories([]))
+	}, [dispatch])
 
-  const dispatch = useAppDispatch();
-  const categories = useSelector(categoriesSelector);
-  const status = useSelector(statusSelector);
-  useEffect(() => {
-    dispatch(fetchCategories([]))
-  }, [dispatch]);
-
-  return (
-        <CommonPageContainer>
-            <Main>
-                <LearnMoreBanner bannerHeading="Madagascars peer-to-peer e-commerce platform" bannerDescription = "Purchase high-quality products made by the people that sell them. By cutting out middlemen, you know exactly where your purchase is from and how it was made." bannerImage={ImageBanner} />
-                <TopCategories categories={categories} status={status} heading="Top Categories"/>
-                <OfferList>
-                  {offers.map(offer => {
-                    return <Offer key={offer.id} 
-                      imageForOffer ={Image}
-                      imageDescription = {offer.imageDescription}
-                      imageForRating=''
-                      offerName= {offer.offerName}
-                      currency= {offer.currency}
-                      amount={offer.amount}
-                      unit= {offer.unit}
-                      star= {offer.star}
-                      ratingDescription = {offer.ratingDescription}
-                      amountOfProduct={offer.amountOfProduct}
-                      offerDescription={offer.offerDescription}
-                      isLearnEnabled={offer.isLearnEnabled}
-                      isFavourited= {offer.isFavourited}
-                      detailButtonText= {offer.detailButtonText}
-                      favoriteButtonText= {offer.favoriteButtonText}
-                      profile= {offer.profile}
-                      name={offer.name}
-                      location={offer.location}/>
-                    })}
-                </OfferList>
-                <CallToAction title="Discover amazing products and profit from a truly fair market:" subTitle="This is a subtitle that is very informative"/>
-            </Main>
-        </CommonPageContainer>
-  )
+	return (
+		<CommonPageContainer>
+			<Main>
+				<LearnMoreBanner
+					bannerHeading='Madagascars peer-to-peer e-commerce platform'
+					bannerDescription='Purchase high-quality products made by the people that sell them. By cutting out middlemen, you know exactly where your purchase is from and how it was made.'
+					bannerImage={ImageBanner}
+				/>
+				<CategoryLabel>Top Categories</CategoryLabel>
+				{status === 'loading' ? (
+					<PendingIndicator alt='PendingSmallIcon' size='s' />
+				) : (
+					<CategoryList
+						categories={categories}
+						primary={true}
+						selectCategory={() => {}}
+					/>
+				)}
+				<Separator />
+				<OffersPage />
```

Comment:
OffersPage is not a page it is a component that should accept offer list as a prop